core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -3950,6 +3950,24 @@ public void checkOrderBy(final boolean desc,
         .returnsUnordered("empid=150; name=Sebastian");
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6904">[CALCITE-6904]
+   * IS_NOT_DISTINCT_FROM is converted error in EnumerableJoinRule</a>. */
+  @Test void testIsNotDistinctFrom() {
```

Comment:
Can you also add a quidem test in one .iq file?

Reply:
Done